dotnet/src/SemanticKernel.Core/Functions/SKFunctionFromPrompt.cs

```diff
@@ -181,11 +177,37 @@ internal sealed class KernelFunctionFromPrompt : KernelFunction
         }
         catch (Exception ex) when (!ex.IsCriticalException())
         {
-            this._logger?.LogError(ex, "Semantic function {Name} execution failed with error {Error}", this.Name, ex.Message);
+            this._logger?.LogError(ex, "Prompt function {Name} execution failed with error {Error}", this.Name, ex.Message);
             throw;
         }
     }
 
+    protected override async IAsyncEnumerable<T> InvokeCoreStreamingAsync<T>(
+        Kernel kernel,
+        SKContext context,
+        AIRequestSettings? requestSettings = null,
+        [EnumeratorCancellation] CancellationToken cancellationToken = default)
+    {
+        var (invokingEventArgs, renderedPrompt, textCompletion, defaultRequestSettings) = await this.PrepareInvokeAsync(kernel, context, requestSettings, cancellationToken).ConfigureAwait(false);
+        if (invokingEventArgs.IsSkipRequested || invokingEventArgs.CancelToken.IsCancellationRequested)
+        {
+            yield break;
+        }
+
+        StringBuilder fullCompletion = new();
+        await foreach (T genericChunk in textCompletion.GetStreamingContentAsync<T>(renderedPrompt, requestSettings ?? defaultRequestSettings, cancellationToken))
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+            fullCompletion.Append(genericChunk);
+        }
+
+        // Update the result with the completion
+        context.Variables.Update(fullCompletion.ToString());
```

Comment:
I agree, this was added to maintain some compliancy with the Pipeline behavior, getting the SKContext.Input and passing it to the next, since we are dropping the pipeline for V1 we may stop doing this as SKContext will also go away.

Reply:
Removed support to Invoked and Logging after streaming